conan/internal/paths.py

```diff
@@ -89,3 +89,5 @@ def _user_home_from_conanrc_file():
 EXPORT_TGZ_NAME = "conan_export.tgz"
 EXPORT_SOURCES_TGZ_NAME = "conan_sources.tgz"
 DATA_YML = "conandata.yml"
+WORKSPACE_YML = "conanws.yml"
```

Comment:
This file is a bit legacy, filenames can be declared directly in conan/api/subapi/workspace.py or other workspace related file